GiniVision/Classes/Core/Multipage Review/MultipageReviewPagesCollectionFooter.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  MultipageReviewPagesCollectionFooter.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 4/26/18.
+//
+
+import Foundation
+
+final class MultipageReviewPagesCollectionFooter: UICollectionReusableView {
+    
+    static let identifier = "MultipageReviewPagesCollectionFooterIdentifier"
+    var didTapAddButton: (() -> Void)?
```

Comment:
Again, you are mixing delegates and callbacks in the same project. You are missing a little consistency. But it's ok, I guess.